middlewares/validate.js

```diff
@@ -0,0 +1,11 @@
+import {  buildCheckFunction } from 'express-validator';
+
+const checkBodyAndQuery = buildCheckFunction(['body', 'params', 'query']);
+
+export const validParamId = [
+    checkBodyAndQuery('id')
+      .optional({ nullable: true })
```

Comment:
check nullable. It should be false

Reply:
Test by sending a postman request that doesn't have an id in the URL.
Also add an error message with `withMessage` that says that `id is missing`

Reply:
Since i ran the migration with "allowNull: false" for the "id", it automatically returns an error when trying to get meals without an "id"

Also the method is "getElementById" and route is "/meals/:id" so an error automatically returns when the URL does not include an "id" i.e when it is http://localhost:3000/meals/(NO id)

Reply:
Also, I noticed the "optional()" field does not accept "withMessage" error message
i don't understand why